crates/wysiwyg/src/dom/unicode_string.rs

```diff
@@ -132,6 +149,11 @@ impl<S: UnicodeString> UnicodeStringExt for S {
 pub trait UnicodeStrExt: UnicodeStr {
     fn is_empty(&self) -> bool;
     fn len(&self) -> usize;
+    fn find_graphemes_at(
+        &self,
+        index: usize,
+    ) -> (Option<String>, Option<String>);
```

Comment:
Sorry, one last thing! Can this return `Self`, so we do the conversion back to S inside the implementation?